google-cloud-storage/samples/acceptance/retry_test.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper"
+require_relative "../storage_configure_retries"
+
+describe "Retry Samples" do
+  let(:local_file) { File.expand_path "data/file.txt", __dir__ }
+  let(:remote_file_name) { "path/file_name_#{SecureRandom.hex}.txt" }
+  let(:bucket) { @bucket }
+
+  before :all do
```

Comment:
Please remove the `:all` on the `before` and `after` clauses. It doesn't do anything, and serves to confuse people because in RSpec (a different test framework) it _does_ have an effect.